src/views/QuestionView.vue

```diff
@@ -242,16 +238,33 @@ export default {
             }
           });
           if (
-            result.data.questions.length < count &&
+            result.data.count < count &&
             (clean || !this.questionBuffer.includes(NO_QUESTION_LEFT))
           ) {
             dataToAdd.push(NO_QUESTION_LEFT);
           }
+          if (this.page == Math.ceil(this.remainingQuestionCount / count)) {
+            if (!this.questionBuffer.includes(NO_QUESTION_LEFT)) {
+              if (clean) {
+                this.questionBuffer = [NO_QUESTION_LEFT];
+              } else {
+                this.questionBuffer = [...this.questionBuffer, ...dataToAdd];
+                this.questionBuffer = [
+                  ...this.questionBuffer,
+                  NO_QUESTION_LEFT,
+                ];
+              }
+            }
+            return;
+          }
           if (clean) {
             this.questionBuffer = [...dataToAdd];
           } else {
             this.questionBuffer = [...this.questionBuffer, ...dataToAdd];
           }
+          if(this.questionBuffer.length==0 && sortBy=="popular"){
```

Comment:
Maybe more `if(dataToAdd.length === 0)`

If This page has nothing to add let see the next one
Instead of "If my buffer is empty let load a new page"